keyboards/lfkeyboards/lfk87/rules.mk

```diff
@@ -6,11 +6,6 @@
 #
 LFK_REV = C
 
-ifeq ($(LFK_REV), A)
```

Comment:
Not entirely sure if this will work because bootloader.mk needs `MCU` to be set before processing `BOOTLOADER`.